go.mod

```diff
@@ -50,7 +51,7 @@ require (
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e // indirect
 	golang.org/x/sys v0.0.0-20191110163157-d32e6e3b99c4 // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	google.golang.org/api v0.10.0 // indirect
+	google.golang.org/api v0.10.0
```

Comment:
why is this not `indirect` anymore ? same for `cloud.google.com/go/storage` above, I don't see any imports in our code for those, shouldn't this mean they should be `indirect` ?

Reply:
I have no idea. I just build and commit whatever Go thinks is right. 🤷‍♂️